frontend/src/views/applications/ApplicationsHistoryView.vue

```diff
@@ -235,7 +235,9 @@ export default {
       }
     },
     isAddCoreApplicationAllowed() {
-      return this.userInfo?.facilities?.some((facility) => facility.intakeWindowCheckForAddApplication && facility.ccofEnrolmentCheckForAddApplication)
+      return this.userInfo?.facilities?.some(
```

Comment:
would it be worth adding a check here for a re-directed application? The facility is still marked as active - but they shouldn't be allowed to start another OFM core app

Reply:
thanks Jen! it's updated